### DISCUSSION TITLE
Only hide certain channel points redeems

### DISCUSSION THREAD

**Discussion by offron** [Chatterino/chatterino2#4587] [Q&A] [answered]
Basicly there are channel points that can be redeemed for a jumpscare.

Problem here is, that the message of redeeming shows a few seconds before in chat, 
making them ineffective when the streamer is reading the chat all the time.

So we are looking for a way of hidding only the redeem message or worst case if otherwise not possible, all of them.
The other messges should stay uneffected.

I tried to to set filters with the flags: `flags.points_redeemed` and `(flags.points_redeemed) contains ("redeemd")` seperatly. 
Activating those, wont show any messages atl all anymore in chat. Not even the point redeems.

Do i have the wrong filter flags? Am in the wrong setting?

Btw, blocking the words, doesnt effect the message, plus it only replaces it with *** / or others, which is giving it away aswell that its a jumpscare.

**Comment by Felanbird** [comment, marked as answer]
You can use `!(flags.reward_message && (message.content contains "nameOfRedemption"))` if the redeem in question does _not_ allow user input, which I assume a jumpscare redemption would not.

If the redeem _does_ include user content then you won't able to limit only that redemption, as `message.content` will be busy dealing with the message content. In that case just use `!flags.reward_message`

`flags.points_redeemed` is for the built-in Twitch redemption that highlights your message.

**Reply by offron** [reply]
worked flawless. Thank you so much!
